static/css/devhub/base.less

```diff
@@ -62,21 +65,82 @@ section {
   }
 }
 
-// Components
+.content {
+  width: 100%;
+
+  > div {
+    color: @color-text;
+    padding: 20px 20px;
+
+    @media @tablet {
+      padding: 20px 60px;
+      max-width: @grid-max;
+      margin: 0 auto;
+    }
+  }
+
+  p,
+  li {
+    font-weight: 300;
+    color: @color-text;
+  }
+}
+
+.scheme-light {
+  background-color: @color-scheme-light-background;
+
+  p,
+  li {
+    color: @color-scheme-light-text;
+  }
+
+  &:before,
```

Comment:
Actually, most of them aren't in-use anymore and I'll remove them. Forgot about it. But good point, don't know why I forgot the colon everywhere.